account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/MFAMethodsPutHandler.java

```diff
@@ -172,12 +189,31 @@ public APIGatewayProxyResponseEvent updateMFAMethodsHandler(
             return generateApiGatewayProxyErrorResponse(500, ErrorResponse.ERROR_1071);
         }
 
-        if (updateResult.updateTypeIdentifier() != null) {
-            var emailNotificationType =
-                    mapEmailNotificationIdentifierToType(updateResult.updateTypeIdentifier());
+        var updateTypeIdentifier = updateResult.updateTypeIdentifier();
+
+        if (updateTypeIdentifier != null) {
+            var emailNotificationType = mapEmailNotificationIdentifierToType(updateTypeIdentifier);
 
             sendEmailNotification(
                     emailNotificationType, putRequest.userProfile.getEmail(), userLanguage);
+
+            var postUpdateDefaultMfaMethod =
+                    successfulUpdateMethods.stream()
+                            .filter(mfaMethod -> DEFAULT.name().equals(mfaMethod.getPriority()))
+                            .findFirst()
+                            .orElseThrow();
+
+            var maybeAuditEventStatus =
+                    sendAuditEvent(
+                            AUTH_MFA_METHOD_SWITCH_COMPLETED,
```

Comment:
This reads to me that we alway send the SWITCH_COMPLETED event here, regardless if it is a switch.

Could we move the `if (updateTypeIdentifier == MFAMethodUpdateIdentifier.SWITCHED_MFA_METHODS)` within `sendAuditEvent` to wrap here instead?

Thanks